contracts/token/ERC721/extensions/ERC721Enumerable.sol

```diff
@@ -87,6 +87,48 @@ abstract contract ERC721Enumerable is ERC721, IERC721Enumerable {
         }
     }
 
+    /**
+     * @dev Hook that is called before any batch token transfer. For now this is limited
+     * to batch minting by the {ERC721Consecutive} extension.
+     *
+     * Calling conditions:
+     *
+     * - When `from` and `to` are both non-zero, ``from``'s `tokenId` will be
+     * transferred to `to`.
+     * - When `from` is zero, `tokenId` will be minted for `to`.
+     * - When `to` is zero, ``from``'s `tokenId` will be burned.
+     * - `from` cannot be the zero address.
+     * - `to` cannot be the zero address.
+     *
+     * To learn more about hooks, head to xref:ROOT:extending-contracts.adoc#using-hooks[Using Hooks].
+     */
+    function _beforeConsecutiveTokenTransfer(
```

Comment:
This really defeats the purpose of using consecutive transfers to begin with... I think this should just revert.

Reply:
To me we have to options:
- revert, and just prevent anyone from using the two extensions together
- (strongly) discourage the two being used ... but make sure that things work as much as we can if devs go against our recommandations.

Reply:
Personally I'd go with revert because the revert would be triggered at constructor time. Otherwise it's going to be unwarranted "OpenZeppelin makes everything expensive" all over again.